### PR TITLE
Rebrand + Restyle

### PR DESCRIPTION
**Pull request by emilyJLin95**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

## Impacted parts

For content changes:

- [x] Documentation content
- [x] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [x] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/use-linea/fund.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ description: Use a faucet to obtain test ETH and USDC
...
+Use [MetaMask](https://docs.metamask.io/guide/getting-started.html) to store your funds and interact with Linea. You can [install MetaMask as a browser extension or mobile app](https://metamask.io/download/).
 
 ## Get test ETH on Goerli
 
 Use a faucet to obtain Goerli ETH. Transactions are much cheaper than Mainnet. Therefore 0.1 ETH is enough to execute a basic workflow. You can [view a list of faucets](https://faucetlink.to/goerli) for the Goerli network.
 
 ## Get test USDC on Goerli
 
-To obtain test USDC on Goerli, use our [USDC faucet](https://bridge.goerli.zkevm.consensys.net/faucet?token=USDC) to mint 10 USDC. You must have some Goerli test ETH to pay for the transaction to mint USDC.
+To obtain test USDC on Goerli, use our [USDC faucet](https://bridge.goerli.linea.build/faucet?token=USDC) to mint 10 USDC. You must have some Goerli test ETH to pay for the transaction to mint USDC.
```
Comment (jtbeckman) [inline docs/use-linea/fund.md:15]: The bridge domain has not been migrated over yet. I brought it up earlier today in Slack https://consensys.slack.com/archives/C04ATK1LXDM/p1679679985475689. 

CC @julien-marchand

Comment (emilyJLin95) [reply docs/use-linea/fund.md:15]: Ah yeah I was wondering if it would be migrated by the due date

Comment (emilyJLin95) [reply docs/use-linea/fund.md:15]: If not I can roll back this change

File: docs/reference/status.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+description: Get the status of the Linea network
 sidebar_position: 1
 ---
 
 # Get the network status
 
-To find the status of ConsenSys zkEVM network, you'll need to:
+To find the status of the Linea network, you'll need to:
 
 1. Navigate to the [Infura status page](https://status.infura.io/).
```
Comment (bgravenorst) [inline docs/reference/status.md:11]: Is this still accurate that users need to check the Infura page for the network status?

**Review by bgravenorst — APPROVED** [review]
LGTM, I added an announcement bar to the top of the site in case users enter the website via an old link.

And there's just one additional question regarding the network status.

File: docs/reference/status.md (hunk 2 above)
Comment (emilyJLin95) [reply docs/reference/status.md:11]: yep

**Comment by emilyJLin95** [comment]
Did a temporary theme change while we wait on official designs

FYI @bgravenorst 

https://user-images.githubusercontent.com/17280653/228002706-de6395c1-d143-4a85-ad0d-a3c2931bbd03.mov

File: docs/developers/ethereum-differences.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ sidebar_position: 3
 
 All EVM opcodes are managed identical to Ethereum unless presented in the following table. This table will be updated following each release.
 
-| Opcode Name                 | ConsenSys zkEVM        |
+| Opcode Name                 | Linea                  |
```
Comment (bgravenorst) [inline docs/developers/ethereum-differences.md:12]: topic title should use sentence case to be consistent with the rest of the docs.

**Review by jlazoff — APPROVED** [review]

**Pull request merged**